andela/eventx
# Ft event dashboard 129621349

# [129621349] As staff and volunteers I get access to my own dashboard to view and manage all my events
#### What does this PR do?

Implement event management for all event staff
#### Description of Task to be completed?

Implement feature that allows staff who have been added to an event to access some event management actions based on their role.
#### How should this be manually tested?

Visit the dashboard. If the user has been added as a staff to an event team, "My Events Teams" should show up in the sidebar. Clicking this link would show a list of all events the user has been added to. User would also see the actions available to them.
#### What are the relevant pivotal tracker stories?
#129621349
#### Screenshots (if appropriate)

![screen shot 2016-10-10 at 6 55 15 pm](https://cloud.githubusercontent.com/assets/20647322/19245830/3214660c-8f1b-11e6-9860-e38a9ef9ed99.png)
#### Questions:
